### ISSUE TITLE
Allow https host

### ISSUE THREAD

**Issue by yonizaf** [ludviglundgren/qbittorrent-cli#71]
I need to be able to add torrents to an instance of qBittorrent WebUI that is set to work through https.

I could not find such a setting. The setting for 'host' takes domain or IP (not URL), then tries to connect over http. 

Is there a configuration option that I missed? or is it possible to add the option?

Thanks.

**Comment by BaukeZwart** [comment]
What have you tried?
I have qbit-cli communicating with a remote (HDB) seedbox over https.
Took me a while to figure it out, but works.
I'm running a build I made using the latest dev branch, not sure if that makes a difference.

**Comment by yonizaf** [comment]
I just added `host = "whatever.com"` to the config file. when running `qbt list` I got an error:
`login error: Post "http://whatever.com:8082/api/v2/auth/login": EOF`

I tried to add `"https://whatever.com"`, but it just tried `"http://https://whatever.com"`, so no help there.

Not sure what I could try further, couldn't find anything else in the documentation or elsewhere.

**Comment by BaukeZwart** [comment]
This what I use, but it's using dev build, I think the variable naming changed.
```addr      = "https://XXX.dedi.hostingby.design/qbittorrent"
login     = "REDACTED"
password  = "REDACTED"
basicUser = "REDACTED"
basicPass = "REDACTED"```

**Comment by ludviglundgren** [comment]
Hi! The project is in the middle of a pretty major refactor with a lot of improvements, fixes and changes. More to come!

What BaukeZwart posted above should work.

**Comment by ludviglundgren** [comment]
I released a new major version earlier today, `v2.0.0-beta1`.

It's got a lot of changes so check out the README. The help commands are IMO pretty self-explanatory now.
